README.md

```diff
@@ -43,9 +43,11 @@ Supported features of the specification:
 | - [Jakarta Restful WS](http/restful-ws) |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
 |          - [Basic](http/basic)          |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
 |           - [Spring](spring)            |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|           - [http4k][http4k]<sup>†</sup>           |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
+|           - [http4k][http4k]<sup>†</sup>|                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
 |            JSON Event Format            |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
 |    - [Jackson](formats/json-jackson)    |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
+|            Protobuf Event Format        |                          :x:                          |                  :heavy_check_mark:                   |
```

Comment:
@slinkydeveloper , I added these to the `v1.0` columns, but I think technically protobuf became supported in the v1.0.1 spec. I'm guessing that you all don't want another column for each minor version, but feel free to tell me if I am wrong there.

Reply:
Nah this column is fine, sdks doesn't make any distinction between 1.0 and 1.0.1

Reply:
👍 

@JemDay also added support for the proto format to work with version 0.3, so I need to go back and update this to indicate that.